include/swift/AST/DiagnosticsSema.def

```diff
@@ -2544,9 +2544,9 @@ ERROR(retroactive_not_in_extension_inheritance_clause,none,
       "'retroactive' attribute only applies in inheritance clauses in "
       "extensions", ())
 
-ERROR(retroactive_attr_does_not_apply,none,
-      "'retroactive' attribute does not apply; %0 is declared in this module",
-      (Identifier))
+WARNING(retroactive_attr_does_not_apply,none,
```

Comment:
This is specified in the proposal to be an error. How about we leave it as an error and use `warnUntilSwiftVersion(6)` to stage it in?

Reply:
Done